.github/workflows/ci-home.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           which ruby
           which irb
           which ssh
+          zsh -c 'echo "$PATH"'
```

Comment:
MacOS 14

Before - https://github.com/kachick/dotfiles/actions/runs/8400851642/job/23008578228?pr=509
---

```
/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/opt/homebrew/lib/ruby/gems/3.0.0/bin:/opt/homebrew/opt/ruby@3.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/runner/.cargo/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Library/Frameworks/Python.framework/Versions/Current/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.dotnet/tools:/Users/runner/.local/share/tmpbin
```

Reply:
After - https://github.com/kachick/dotfiles/actions/runs/8400967389/job/23008835548?pr=509
---

```
/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/opt/homebrew/lib/ruby/gems/3.0.0/bin:/opt/homebrew/opt/ruby@3.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/runner/.cargo/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Library/Frameworks/Python.framework/Versions/Current/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.dotnet/tools:/Users/runner/.local/share/tmpbin
```

Reply:
After https://github.com/kachick/dotfiles/pull/509/commits/9352fc511a7bf12fb735cb7f3aa7b8f822d8acff - https://github.com/kachick/dotfiles/actions/runs/8400999541/job/23008911703?pr=509
---

```
/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/Users/runner/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/opt/homebrew/lib/ruby/gems/3.0.0/bin:/opt/homebrew/opt/ruby@3.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/runner/.cargo/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Library/Frameworks/Python.framework/Versions/Current/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.dotnet/tools:/Users/runner/.local/share/tmpbin
```

Reply:
Ah, not changed at here...

above printing shell paths, that shows. But bash is still broken...

---

In local zsh

before

```console
↕  3 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which bash
/nix/store/d5z82b7rgqklrl4yycn44npi5qmpzzly-bash-interactive-5.2p26/bin/bash

↕  3 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which zsh
/bin/zsh

↕  3 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which fish
/Users/kachick/.nix-profile/bin/fish

```

after

```console
↕  4 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which zsh
/Users/kachick/.nix-profile/bin/zsh

↕  4 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which bash
/Users/kachick/.nix-profile/bin/bash

↕  4 dotfiles kill-path_helper(9352fc5)  ≡via 🐹 v1.22.1 via ❄  impure (nix-shell-env) zsh
> which fish
/Users/kachick/.nix-profile/bin/fish
```

Reply:
```
> bat /etc/profile
───────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /etc/profile
───────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ # System-wide .profile for sh(1)
   2   │
   3   │ if [ -x /usr/libexec/path_helper ]; then
   4   │     eval `/usr/libexec/path_helper -s`
   5   │ fi
   6   │
   7   │ if [ "${BASH-no}" != "no" ]; then
   8   │     [ -r /etc/bashrc ] && . /etc/bashrc
   9   │ fi
───────┴──────────────────────────────────────────
```